README.md

```diff
@@ -4002,6 +4002,44 @@ _You can enable the following settings in Xcode by running [this script](resourc
   ```
 
   </details>
+  
+* <a id='unused-private-declaration'></a>(<a href='#unused-private-declaration'>link</a>) **Remove unused private and fileprivate properties, functions, and typealiases** [![SwiftFormat: unusedPrivateDeclaration](https://img.shields.io/badge/SwiftFormat-unusedPrivateDeclaration-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#unusedPrivateDeclaration)
+
+  <details>
+
+  #### Why?
+  
+  Improves readability since the code has no effect and should be removed for clarity.
+  
+  ```swift
+  // WRONG: Unused private property is unused and redundant
+  struct Planet {
+    private var mass: Double
+    
+    var age: Double
+  }
+
+  // WRONG: Unused private function is unused and redundant
+  struct Planet {
+    private func distance(to: Planet) { }
+    
+    var age: Double
+  }
+
+  // WRONG. Unused private typealias is unused and redundant
+  struct Planet {
+    private typealias Dependencies = UniverseBuilderProviding
+    
+    var age: Double
+  }
+    
+  // RIGHT
+  struct Planet {
+    var age: Double
+  }
```

Comment:
We could consolidate these three examples into a single example for brevity. Shorter examples are usually easier for folks to understand.

Also, maybe the example could include a private declaration that _isn't_ unused.

```suggestion
  // WRONG: Includes private declarations that are unused
  struct Planet {
    var ageInBillionYears: Double {
      ageInMillionYears / 1000
    }
    
    private var ageInMillionsOfYears: Double
    private typealias Dependencies = UniverseBuilderProviding // unused
    private var mass: Double // unused
    private func distance(to: Planet) { } // unused
  }
    
  // RIGHT
  struct Planet {
    var ageInBillionsOfYears: Double {
      ageInMillionYears / 1000
    }
    
    private var ageInMillionYears: Double
  }
```

Reply:
Added!